.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  checkout_cuberite:
+    working_directory: ~/cuberite
+    docker:
+    - image: peterbell10/cuberite-circleci:0.0.2
+    steps:
+    - checkout
+    - run: git submodule update --init
+    - save_cache:
+        key: "cuberite-{{ .Environment.CIRCLE_SHA1 }}"
+        paths:
+        - ~/cuberite
```

Comment:
Just recently read on [Travis CI](https://docs.travis-ci.com/user/caching/):

> Large files that are quick to install but slow to download do not benefit from caching, as they take as long to download from the cache as from the original source:

Did you check, if caching leads to any speed improvements?

Reply:
And how about the packaging and uploading? If it's below ~10s, I'm fine. Otherwise checking out both would be faster.

Reply:
It's ~8s.